packages/sharec-schema/lib/actions/compose.js

```diff
@@ -27,13 +27,12 @@ const applySchemaByKeys = ({ schema, keys = [], target }) =>
     for (const key of keys) {
       const strategy = schema[key] || schema.$$default
       const ignoreList = schema.$$ignore || []
+      const current = params.current.get(key)
+      const notChanged = !strategy || !params.upcoming.has(key) || ignoreList.includes(key)
 
-      if (!strategy || !params.upcoming.has(key) || ignoreList.includes(key)) {
-        target.set(key, params.current.get(key))
-        continue
-      }
+      if (notChanged && current !== undefined) target.set(key, current)
```

Comment:
Add `continue` here, to prevent next expression call. I know, that it doesn't have any sense to do that, but in future, we can expand `notChanged` behaviour and just forgot that this one doesn't breaks current iteration